epuck_ros2_driver/src/vl53l0x/LICENSE.txt

```diff
@@ -0,0 +1,71 @@
+Copyright (c) 2017 Pololu Corporation.  For more information, see
```

Comment:
All these files were included as a sub-module before instead of copying them, is there a reason for not using sub-modules anymore?

Reply:
You are right. Git got confused after renaming. Fixed now

Reply:
Indeed, git handles badly submodule renaming.
Thank you for the fix.